WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingCreateLabelsViewModel.swift

```diff
@@ -331,7 +332,7 @@ private extension WooShippingCreateLabelsViewModel {
             guard let self else { return }
             switch result {
             case .success(let address):
-                destinationAddress = address.normalizedAddress
+                destinationAddress = address.normalizedAddress.toWooShippingAddress()
```

Comment:
I ran into an issue after confirming the edited destination address: the change wasn't reflected in the bottom sheet UI.

I realized that we're updating the `destinationAddress` property here but the UI displays `destinationAddressLines`, which is a lazy variable that isn't updated at this point. We could change that to a computed variable so it follows any changes to `destinationAddress`.

Reply:
Nice catch and suggestion. Done in https://github.com/woocommerce/woocommerce-ios/pull/15260/commits/a570b8a68f593c22cda6285e29cf7e0d7eed2b51 Thanks!